gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Jun 08 21:11:25 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
Why not 4.8 or whatever is latest?

Reply:
I just had Android Studio apply it's automatic changes, and I think it's just going to the lowest possible version, but not sure.

Reply:
Let's check if it works with 4.8 and move to that.

Reply:
@AudriusButkevicius Seems that Android Studio is not too happy with Gradle >= 4.6:
```
Configuration on demand is not supported by the current version of the Android Gradle plugin since you are using Gradle version 4.6 or above. Suggestion: disable configuration on demand by setting org.gradle.configureondemand=false in your gradle.properties file or use a Gradle version less than 4.6.
```
I'll try with 4.5 instead.